webpack/components/extensions/Hosts/BulkActions/BulkRepositorySetsWizard/01_BulkRepositorySetsTable.js

```diff
@@ -0,0 +1,395 @@
+                key="split-action"
+                aria-label="Action"
+                onClick={() => handleActionToggle(actionDropdownValue)}
+              >
+                {dropdownValues[actionDropdownValue]}
+              </MenuToggleAction>],
+          }}
+          aria-label="Content override action to apply to all selected repositories"
+          isDisabled={selectedCount === 0}
+        />
+      )}
+    >
+      <DropdownList>
+        {Object.entries(dropdownValues).map(([key, value]) => (
+          <DropdownItem
+            key={key}
+            value={key}
+            ouiaId={`content-override-action-dropdown-${key}`}
+            onClick={() => setActionDropdownValue(key)}
+          >
+            {value}
+          </DropdownItem>
+        ))}
+      </DropdownList>
+    </Dropdown>
+  );
+
+  const [columnNamesKeys, keysToColumnNames] = getColumnHelpers(columns);
+
+  return (
+    <>
+      {shouldValidateStep1 && !repoSetsSelectionIsValid && (
+        <Alert
+          ouiaId="no-reposet-changes-alert"
+          variant="info"
+          isInline
+          title={__('Change the status of at least one repository.')}
+          style={{ marginBottom: '1rem' }}
+        />
+      )}
+      <TableIndexPage
+        customToolbarItems={[actionButton]}
+        selectionToolbar={selectionToolbar}
+        results={repoSetsResults}
+        metadata={repoSetsMetadata}
+        response={repoSetsResponse}
+        tableType="repository_sets"
+        apiUrl={REPO_SETS_URL}
+        apiOptions={apiOptions}
+        selectedCount={selectedCount}
+        selectPage={selectPage}
+        selectNone={selectNone}
+        customSearchProps={{
+          autocomplete: {
+            url: katelloApi.getApiUrl('/repository_sets/auto_complete_search'),
+          },
+        }}
+        bulkSelect={repoSetsBulkSelect}
+        topPagination={<Pagination
+          key="table-index-page-top-pagination"
+          updateParamsByUrl={false}
+          variant={PaginationVariant.top}
+          page={page}
+          perPage={perPage}
+          itemCount={subtotal}
+          onChange={onPagination}
+        />}
+      >
+        <Table
+          childrenOutsideTbody
+          showCheckboxes
+          rowSelectTd={RowSelectTd}
+          selectOne={selectOne}
+          isSelected={isSelected}
+          isEmbedded
+          idColumn="label"
+          columns={columns}
+          refreshData={noop}
+          url=""
+          results={repoSetsResults}
+          isPending={repoSetsLoadingStatus === STATUS.PENDING}
+          params={repoSetsParams}
+          setParams={setRepoSetsParamsAndAPI}
+          bottomPagination={
+            <Pagination
+              key="table-bottom-pagination-yes-you-better"
+              page={page}
+              perPage={perPage}
+              itemCount={subtotal}
+              onChange={onPagination}
+              updateParamsByUrl={false}
+            />
+          }
```

Comment:
Instead of bottomPagination this can just be:
```
  page={page}
  perPage={perPage}
  itemCount={subtotal}
```
and change params to `params={{ ...repoSetsMetadata, perPage, page}}`